eZ/Publish/Core/Pagination/Pagerfanta/ContentSearchHitAdapter.php

```diff
@@ -65,7 +77,7 @@ public function getSlice($offset, $length)
         $query->limit = $length;
         $query->performCount = false;
 
-        $searchResult = $this->searchService->findContent($query);
+        $searchResult = $this->searchService->findContent($query, $this->languageFilter);
```

Comment:
Correct me if I'm wrong, but it seems that we are missing an assertion ensuring that `language Filter` is passed to `findContent` ContentSearchHitAdapterTest.